.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-    - 1.7
+    - 1.9
```

Comment:
Would you mind submitting this version bump separately to keep the commit log clean?